.ruby-version

```diff
@@ -1 +1 @@
-2.3.1
+ruby-2.4.1
```

Comment:
Yes, but this file is also used by `rbenv` and so it should really just contain the version numbers.  (The production system uses `rbenv`, BTW.)
You can have RVM put just the version numbers in there or you can create it manually. 
(Note that the content you're replacing only has the version numbers in it.)